src/store/audio.ts

```diff
@@ -71,58 +71,52 @@ function parseTextFile(
   defaultStyleIds: DefaultStyleId[],
   userOrderedCharacterInfos: CharacterInfo[]
 ): AudioItem[] {
-  const characters = new Map<string, { engineId: string; styleId: number }>();
-  const uuid2StyleIds = new Map<
-    string,
-    { engineId: string; styleId: number }
-  >();
+  const characters = new Map<string, Voice>();
+  const uuid2Voice = new Map<string, Voice>();
```

Comment:
ここの関数変えてくださってるの助かります！！！
ちょっとご面倒なお願いかもなのですが、ついでにこの２変数名前ややこしいので変えちゃうのとかどうでしょう･･････！！！
（string型になっててなかなか難しく。。）

```suggestion
  const speakerNameToVoice = new Map<string, Voice>();
  const speakerIdToVoice = new Map<string, Voice>();
```
あるいは
```suggestion
  const nameToVoice = new Map<string, Voice>();
  const uuidToVoice = new Map<string, Voice>();
```

Reply:
name2Voiceへ変更しました